open-company/open-company-web
# Homepage redirect

Card: https://trello.com/c/edD7WE0z

Item:
> Send user into digest from /, not to home page

Redirect user from / to /{last-org}/all-posts if he's logged in and there is no no_redirect parameter in the url. The parameter is used to let the user navigate the marketing site from within it. All the links going back to homepage from the other marketing site pages should have it now.

To test:
- login
- visit /
- [x] are you redirected to your dashboard? Good
- visit /about or /pricing
- click on the carrot in top left
- [x] are you redirected to our homepage and not to the dashboard? Good
- now logout
- visit /
- [x] do you see our marketing homepage? Good
